.vale/styles/Openverse/TermCasing.yml

```diff
@@ -5,17 +5,19 @@ message: "Incorrect casing. Use '%s' instead of '%s'."
 level: error
 ignorecase: false
 scope:
-  - text
+  # Non-code sentence level text OR headings
+  - summary
+  - heading
```

Comment:
Changes in this file I noticed while writing and editing the documentation for the new package. These should be more reliable and avoid false positives like variable identifiers inside of code blocks (which is what I ran into that prompted me to look into it).